pandas/core/generic.py

```diff
@@ -5661,22 +5661,58 @@ def clip_lower(self, threshold, axis=None, inplace=False):
         """
         Return copy of the input with values below given value(s) truncated.
 
+        It truncates values below a certain threshold. Threshold can be a single
+        value or an array, in the latter case it performs the truncation
+        element-wise.
+
         Parameters
         ----------
         threshold : float or array_like
+            Minimum value allowed. All values below threshold will be
+            set to this value.
         axis : int or string axis name, optional
             Align object with threshold along the given axis.
         inplace : boolean, default False
-            Whether to perform the operation in place on the data
-                .. versionadded:: 0.21.0
+            Whether to perform the operation in place on the data.
 
         See Also
         --------
-        clip
+        clip : Return copy of input with values below and above thresholds truncated.
 
         Returns
         -------
         clipped : same type as input
+
+        Examples
+        --------
+        >>> s = pd.Series([3.816052, 7.969235, 6.903116, 2.532261, 5.015296])
+        >>> s
```

Comment:
I think it'll be much easier for people to understand the example with integers (e.g. `[1, 2, 3, 4]`). Not sure if I'm missing some reason for using this numbers with so many decimals.